### ISSUE TITLE
youtube-dl for Mac?

### ISSUE THREAD

**Issue by TarekSalama** [rg3/youtube-dl#5902]
So I used to use youtube-dl for Windows and I understand how to make it read the options from a config file and download several videos from a text batch file.

I bought a Mac and I've never used one before. I tried my best to mirror the way I used it on Windows but I failed.

Steps I've done so far:

In terminal:
Last login: Fri Jun  5 23:37:38 on ttys000
The-Pentagon:~ tareksalama$ youtube-dl
-bash: /usr/local/bin/youtube-dl: Permission denied
The-Pentagon:~ tareksalama$ sudo youtube-dl
Password:
sudo: youtube-dl: command not found
The-Pentagon:~ tareksalama$ youtube-dl version
-bash: /usr/local/bin/youtube-dl: Permission denied
The-Pentagon:~ tareksalama$ sudo youtube-dl version
sudo: youtube-dl: command not found
The-Pentagon:~ tareksalama$ sudo curl https://yt-dl.org/latest/youtube-dl -o /usr/local/bin/youtube-dl
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  880k  100  880k    0     0   274k      0  0:00:03  0:00:03 --:--:--  274k
The-Pentagon:~ tareksalama$ youtube-dl
-bash: /usr/local/bin/youtube-dl: Permission denied
The-Pentagon:~ tareksalama$ sudo youtube-dl
sudo: youtube-dl: command not found
The-Pentagon:~ tareksalama$ 

So basically, I couldn't even get it to start.

The config file I placed it /Private/etc and named it youtube-dl.config.

Lastly, here's the contents of the config file after changing it to suit OS X:

"-U --no-overwrites --no-cache-dir --restrict-filenames --ignore-errors --batch-file ~/Desktop/batchvideos.txt -o /Volumes/Transcend/%(uploader)s/%(title)s_%(uploader)s_%(id)s.%(ext)s"

So it's supposed to read the links saved inside batchvideos.txt on my desktop and download them into my portable drive (Transcend).

Again, I'm a complete noob on Mac. But those were my attempts.

**Comment by dstftw** [comment]
[https://github.com/rg3/youtube-dl#installation](https://github.com/rg3/youtube-dl#installation).

**Comment by TarekSalama** [comment]
Thanks for your reply.

I did follow the installation document but that's how far I got to. I need a little help.

**Comment by dstftw** [comment]
No, you didn't. I don't see any `chmod` command in your log.
